src/structures/interfaces/TextBasedChannel.js

```diff
@@ -128,7 +128,7 @@ class TextBasedChannel {
    * channel.send({
    *   files: [{
    *     attachment: 'entire/path/to/file.jpg',
-   *     name: 'file.jpg'
+   *     name: 'file.jpg',
    *     description: 'A description of the file'
```

Comment:
```suggestion
   *     description: 'A description of the file',
```

Reply:
There are more places that could use a trailing comma, e.g. after `}]` from files, but they're not set there, and there's a possibility many examples don't even have trailing commas.

Reply:
> There are more places that could use a trailing comma, e.g. after `}]` from files, but they're not set there, and there's a possibility many examples don't even have trailing commas.

So if I'm understanding, it shouldn't be included in this PR, and all examples that could use a trailing comma in a future PR?